### PR TITLE
add isWordRecognizable()

### PR DESCRIPTION
**Pull request by ThThoma**
A function that searches given word and returns the word's symbol code if the word is present in the model's symbol table,
or -1 otherwise.
As explained in the issue #258 , i find this inclusion really helpful.

### CODE REVIEW

File: src/model.cc [hunk 1]
```diff
@@ -295,6 +295,15 @@ void Model::Unref()
     }
 }
 
+int32 Model::IsWordRecognizable(const char *word)
```
Comment (nshmyrev) [inline src/model.cc]: Can you please rename it to something simple like findWord ?

File: src/vosk_api.h [hunk 2]
```diff
@@ -55,6 +55,14 @@ VoskModel *vosk_model_new(const char *model_path);
 void vosk_model_free(VoskModel *model);
 
 
+/** Check if a word can be recognized by the model
+ * @param word: the word
+ * @returns the word symbol if @param word exists inside the model
+ * or -1 otherwise.
+ * Reminding that word symbol 0 is for <epsilon> */
+int IsWordRecognizable(VoskModel *model, const char *word);
```
Comment (nshmyrev) [inline src/vosk_api.h]: You better call it vosk_model_find_word

**Review by nshmyrev — COMMENTED** [review]
Thank you for the pull request!

**Comment by ThThoma** [comment]
I renamed and commited.
Please review.

(I am still learning to use GitHub, so i'm not sure if the commit reached here)

**Comment by nshmyrev** [comment]
Very good, thank you!

**Pull request merged**